lib/Common/ConfigFlagsList.h

```diff
@@ -558,6 +558,8 @@ PHASE(All)
 #define DEFAULT_CONFIG_RegexTracing         (false)
 #define DEFAULT_CONFIG_RegexProfile         (false)
 #define DEFAULT_CONFIG_RegexDebug           (false)
+#define DEFAULT_CONFIG_RegexDebugAST        (true)
+#define DEFAULT_CONFIG_RegexDebugAnnotatedAST (true)
```

Comment:
Should these be true by default?

Reply:
On-by-default for these is the current behavior.

Right now -RegexDebug has 3 "chapters" of output. AST, Annotated AST, and (Bytecode) Program.

Personally I'm usually more interested in reviewing the bytecode program than I am in debugging the AST so I've always wanted to have an option to turn these output segments off (or, at least the Annotated AST, but it was easy to also disable the other chapter while I was in the area). This change adds such flags.

I'd be tempted to turn off the annotated AST which is a ton of output for not a lot of value (unless you're actively working on AST-based optimizations in the regex engine), and would probably vote for leaving basic AST on by default.

Reply:
I see, so these are already behind the RegexDebug flag.

Reply:
Yep!